packages/garbo/src/tasks/yachtzee/outfit.ts

```diff
@@ -88,26 +81,3 @@ export function prepareOutfitAndFamiliar(): void {
     );
   }
 }
-
-export function stickerSetup(expectedYachts: number): void {
```

Comment:
is this not worth it anymore or did this get relocated and I just haven't seen it yet